scaffoldmaker/meshtypes/meshtype_3d_colonsegmentteniacoli1.py

```diff
@@ -195,17 +205,17 @@ def generateMesh(cls, region, options):
         meshrefinement.refineAllElementsCubeStandard3d(refineElementsCountAround, refineElementsCountAlong, refineElementsCountThroughWall)
         return meshrefinement.getAnnotationGroups()
 
-def getColonHaustraSegmentInnerPoints(elementsCountAroundTC, elementsCountAroundHaustrum, elementsCountAlongHaustrum, widthTC, radius,
-    cornerInnerRadiusFactor, haustrumInnerRadiusFactor, haustrumLengthEndDerivativeFactor, haustrumLengthMidDerivativeFactor, haustrumLength):
+def getColonSegmentInnerPoints3TC(region, elementsCountAroundTC, elementsCountAroundHaustrum, elementsCountAlongSegment, widthTC, radius,
```

Comment:
3TeniaColi, or is it going to take number of tenia coli as an argument once you do pig? Can always:
<li>
assert teniaColiCount == 3, 'function name Only implemented for 3 tenia coli'
</li>